CONTRIBUTING.md

```diff
@@ -14,7 +32,7 @@ contributing enhancements to the Loki charm.
   The primary author of this charm is available on the Mattermost channel as
   `@jose-masson`.
 - It is strongly recommended that prior to engaging in any enhancements
-  to this charm you familiarise your self with Juju.
+  to this charm you familiarise your self with [Juju](https://juju.is).
```

Comment:
What do you think if we link to this section? https://juju.is/#what-is-juju

Reply:
Guess it could be phrased in multiple ways. One way to phrase it that I think is a bit more inviting is

> To make sure you get a smooth, worthwhile experience contributing to this charm, we recommend that you first familiarize yourself with Juju. We also recommend that you join the CharmHub Mattermost to get early feedback on your ideas.

Reply:
Throwing another alternative to the mix:

> To author PRs you should know [what is jujuj](https://juju.is/#what-is-juju) and [how operators are written](https://juju.is/docs/sdk). The best way to get a head start is to join the conversation on [our mattermost channel](https://chat.charmhub.io/charmhub/channels/observability)!